### PR TITLE
chore: build core libraries in parallel

### PR DESCRIPTION
**Pull request by Kha**

### CODE REVIEW

**Comment by Kha** [comment]
!bench

**Comment by Kha** [comment]
Red is not nice :laughing:

**Comment by nomeata** [comment]
Oh. eciN.

**Comment by Kha** [comment]
Oh joy, I never actually changed the benchmarks to measure Lake build time as well!?

**Comment by Kha** [comment]
!bench

**Comment by Kha** [comment]
!bench

**Comment by Kha** [comment]
!bench

File: src/stdlib.make.in [hunk 1]
```diff
@@ -33,8 +33,10 @@ endif
 
 # These can be phony since the inner Makefile will have the correct dependencies and avoid rebuilds
 Init:
+	@mkdir -p "${LIB}/lean/Lean" "${LIB}/lean/Lake"
 # Use `+` to use the Make jobserver with `leanmake` for parallelized builds
```
Comment (Kha) [inline src/stdlib.make.in:37]:
```suggestion
# Use `+` to use the Make jobserver with `leanmake` for parallelized builds
# Build .oleans of downstream libraries as well for better parallelism
```

**Pull request merged**